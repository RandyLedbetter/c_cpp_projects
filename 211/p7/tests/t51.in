this_is_a_bad_command these are bad arguments
insert does_your_program_keep_working or_does_it_stop 1
insert does_your_program_keep_working it_should_keep_working_after_a_bad_command 2
lookup does_your_program_keep_working
