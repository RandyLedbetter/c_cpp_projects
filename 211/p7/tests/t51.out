does_your_program_keep_working: none, or_does_it_stop(1), it_should_keep_working_after_a_bad_command(2)
